mantidproject/mantid
# allow one q binning parameter set for D16

**Description of work.**
Allow for a single set of q binning params in `SANSILLAutoProcess`, used on D16.



**To test:**
Add ILL/D16 under your system tests directory to the data search directories.
Then the following should run without errors.
`ws = SANSILLAutoProcess(SampleRuns="3674, 3677, 3680", OutputBinning='0.01')`



Fixes #32347. 


*This does not require release notes as it is a regression introduced in this release*



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
